Resources/Prototypes/Datasets/ion_storm.yml

```diff
@@ -175,7 +175,7 @@
   - GLASS
   - HAPPINESS
   - MEAT
-  - HUMAN CONTACT
+  - HUMANOID CONTACT
```

Comment:
Specifically human contact is overly-specific I think; the alternative would be HUMAN CONTACT, LIZARD CONTACT, MOTH CONTACT etc. Thoughts?